io.github.leolost2605.gradebook.yml

```diff
@@ -0,0 +1,21 @@
+app-id: io.github.leolost2605.gradebook
+
+runtime: io.elementary.Platform
+runtime-version: '7.1'
+sdk: io.elementary.Sdk
```

Comment:
there is no such runtime available on flathub.

Reply:
Sorry didn't realize was just coming from elementary :)
I switched it to gnome

Reply:
tbf I just wish the Elementary people submitted their runtime here.